pkg/rsyslog/init.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+IMGP=$(zboot curpart) /usr/sbin/rsyslogd
```

Comment:
This will have to be just IMGP=IMGA for now

Reply:
Sure. Will change